src/generate/index.ts

```diff
@@ -9,5 +9,31 @@ export { ApiGroup } from "./apiGroup";
 export { ApiCollection } from "./apiCollection";
 export { Template } from "./template";
 
-import * as handlebarsAmfHelpers from "./handlebarsAmfHelpers";
-export { handlebarsAmfHelpers };
+import {
+  getBaseUri,
+  getParameterDataType,
+  getPropertyDataType,
+  getRequestPayloadType,
+  getReturnPayloadType,
+  getProperties,
+  isAdditionalPropertiesAllowed,
+  isOptionalProperty,
+  isRequiredProperty,
+  isTypeDefinition
+} from "./handlebarsAmfHelpers";
+import { getValue } from "./utils";
+const handlebarsAmfHelpers = {
```

Comment:
Why is this being exported as an object instead of what you had before?

Reply:
I misinterpreted a comment and went on a completely different route with getValue and all this. Reverting it.